Gui/siteConfig.py

```diff
@@ -62,12 +62,11 @@
                             "default_hv_voltage": -80,
                             "default_hv_compliance_current": 5e-6,
                             "default_hv_delay": 2,
-                            "default_hv_step_size": 10,
+                            "default_hv_step_size": 10}
                             "relay_board": "RelayBoard",
```

Comment:
I'm surprised this doesn't immediately crash the GUI. I think you can just comment those lines.